demisto_sdk/commands/common/errors.py

```diff
@@ -2542,8 +2562,61 @@ def modeling_rule_keys_are_missing():
         return "The 'rules' key or the 'schema' key is missing from the modeling rule yml file. " \
                'Make sure to add them to your yml file with an empty string as value.'
 
+    @staticmethod
+    @error_code_decorator
+    def modeling_rule_schema_types_invalid(invalid_types: list):
+        return f"The following types in the schema file are invalid {','.join(invalid_types)}. " \
+               f"Valid types are: string, int , float, datetime, boolean."
+
+    @staticmethod
+    @error_code_decorator
+    def modeling_rules_files_naming_error(invalid_files: list):
+        return f"The following modeling rules do not match the naming conventions: {','.join(invalid_files)}. " \
+               f"Files in the modeling rules directory must be titled as the pack."
+
+    @staticmethod
+    @error_code_decorator
+    def correlation_rules_files_naming_error(invalid_files: list):
+        return f"The following correlation rules files do not match the naming conventions: {','.join(invalid_files)}." \
+               f" Files in the modeling rules directory must start with the title of pack."
+
+    @staticmethod
+    @error_code_decorator
+    def xsiam_report_files_naming_error(invalid_files: list):
+        return f"The following xsiam report files do not match the naming conventions: {','.join(invalid_files)}." \
+               f"XSIAM reports title must start with the title of pack."
```

Comment:
```suggestion
        return f"The following xsiam report files do not match the naming conventions: {','.join(invalid_files)}." \
               f"XSIAM reports title must use the pack's name as a prefix, e.g. `myPack-report1.yml`."
```

if this makes sense, please edit otherwise